RadarSDK/RadarAPIClient.m

```diff
@@ -327,10 +331,21 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
                         id eventsObj = res[@"events"];
                         id userObj = res[@"user"];
                         id nearbyGeofencesObj = res[@"nearbyGeofences"];
+                        id tokenObj = res[@"token"];
                         NSArray<RadarEvent *> *events = [RadarEvent eventsFromObject:eventsObj];
                         RadarUser *user = [[RadarUser alloc] initWithObject:userObj];
                         NSArray<RadarGeofence *> *nearbyGeofences = [RadarGeofence geofencesFromObject:nearbyGeofencesObj];
 
+                        if (encrypted) {
+                            if (!tokenObj) {
+                                return completionHandler(status, nil, nil, nil, nil, nil, nil);
+                            }
+
+                            NSString *token = (NSString *)tokenObj;
+
+                            return completionHandler(status, nil, nil, nil, nil, nil, token);
+                        }
```

Comment:
This codepath is completely forgoing updating the event & location update handlers, trips, & place updating.

As such, `trackVerified` and `trackVerifiedEncrypted` behave significantly differently -- wonder how we should make it clear to users the difference in behavior and effects (could see myself getting confused by this).

Reply:
Will mention on /documentation. Most people using this aren't going to be using `startTracking()`, so shouldn't cause unexpected behavior in practice